docs/source/api/apollo-server.md

```diff
@@ -343,15 +343,44 @@ addMockFunctionsToSchema({
    to standard error. Specify this function to process errors in a different
    way.
 
-*  `privateVariables`: Array<String> | boolean
+*  `privateVariables`: Array<String\> | boolean
```

Comment:
Probably put the deprecated one below the new one, and just document it in terms of the new one (ie, say passing a boolean is equivalent to passing safelistAll, etc)